tensorboard/webapp/plugins/npmi/views/annotations_list/annotation/annotation_test.ts

```diff
@@ -74,6 +74,7 @@ describe('Npmi Annotations List Container', () => {
   it('renders annotation', () => {
     const fixture = TestBed.createComponent(AnnotationContainer);
     fixture.componentInstance.activeMetrics = ['nPMI@test', 'nPMI@other'];
+    fixture.componentInstance.runHeight = 30;
```

Comment:
factor the component creation out to a function that takes either data or activeMetrics as input.